website/docs/cloud-docs/workspaces/settings/notifications.mdx

```diff
@@ -20,31 +20,36 @@ To add, edit, or delete notifications for a workspace, go to the workspace and c
+- **Token** (Optional) This is only available for generic webhooks. A token is an arbitrary secret string that Terraform Cloud will use to sign its notification webhooks. Refer to [Notification Authenticity][inpage-hmac] for details. The token is encrypted for storage, so you cannot view it after saving the notification configuration.
+- **Email Recipients** This is only available for emails. Select users that should receive notifications.
 
-   - **Email Recipients** This is only available for emails. Select users that should receive notifications.
+<!-- BEGIN: TFC:only name:drift-detection -->
 
-   - **Workspace Events:** Terraform Cloud can send notifications for all events or only for specific events. The following events are available:
-       - **Drift**: Terraform Cloud detected configuration drift. This notification is only available if [drift detection](/cloud-docs/workspaces/settings/drift-detection) is enabled for the workspace.
-       - **Drift Check Fail**: A drift detection assessment failed. This notification is only available if [drift detection](/cloud-docs/workspaces/settings/drift-detection) is enabled for the workspace.
-   - **Run Events:** Terraform Cloud can send notifications for all events or only for specific events. The following events are available:
-       - **Created**: A run is created and enters the [Pending stage](/cloud-docs/run/states#1-the-pending-stage).
-       - **Planning**: A run acquires the lock and starts to execute.
-       - **Needs Attention**: A plan has changes and Terraform requires user input to continue. This may include approving the plan or a [policy override](/cloud-docs/run/states#7-the-policy-check-stage).
-       - **Applying**: A run enters the [Apply stage](/cloud-docs/run/states#8-the-apply-stage), where Terraform makes the infrastructure changes described in the plan.
-       - **Completed**: A run has completed successfully.
-       - **Errored**: A run has terminated early due to error or cancellation.
+- **Workspace Events:** Terraform Cloud can send notifications for all events or only for specific events. The following events are available:
+  - **Drift**: Terraform Cloud detected configuration drift. This notification is only available if [drift detection](/enterprise/workspaces/settings/drift-detection) is enabled for the workspace.
+  - **Drift Check Fail**: A drift detection assessment failed. This notification is only available if [drift detection](/enterprise/workspaces/settings/drift-detection) is enabled for the workspace.
+
+<!-- END:   TFC:only name:drift-detection -->
+
+- **Run Events:** Terraform Cloud can send notifications for all events or only for specific events. The following events are available:
+  - **Created**: A run is created and enters the [Pending stage](/enterprise/run/states#1-the-pending-stage).
```

Comment:
```suggestion
  - **Created**: A run begins and enters the [Pending stage](/enterprise/run/states#1-the-pending-stage).
```

Suggest changing "A run is created" to "A run begins" to avoid passive voice and keep tense consistency with "enters".

Reply:
yeah this is fair - the only reason I hesitate is that I'm not sure if "created" is more appropriate as the backend probably creates a run object. But you know what? that prob doesn't really matter in this context and your edit is better :p thank you!